src/main/java/installers/FormplayerOfflineUserRestoreInstaller.java

```diff
@@ -46,6 +59,22 @@ public void readExternal(DataInputStream in, PrototypeFactory pf) throws IOExcep
 
     }
 
+    @Override
+    public boolean install(Resource r, ResourceLocation location,
+                           Reference ref, ResourceTable table,
+                           CommCarePlatform platform, boolean upgrade)
+            throws UnresolvedResourceException, UnfullfilledRequirementsException {
+        OfflineUserRestore offlineUserRestore = new OfflineUserRestore();
```

Comment:
Am I following that this just creates an empty/dummy user restore object? That seems fine, but what is the reason that we can't just have this method do nothing?